athena.core/src/main/java/org/catools/athena/pipeline/model/PipelineExecutionDto.java

```diff
@@ -0,0 +1,67 @@
+package org.catools.athena.pipeline.model;
+
+import org.catools.athena.core.model.MetadataDto;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.Accessors;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+
+@Data
+@NoArgsConstructor
+@Accessors(chain = true)
+public class PipelineExecutionDto implements Serializable {
+
+  private Long id;
+
+  @NotBlank
+  @Size(max = 300)
+  private String packageName;
+
+  @NotBlank
+  @Size(max = 100)
+  private String className;
+
+  @NotBlank
+  @Size(max = 100)
+  private String methodName;
```

Comment:
Let's add fields for parameters, for data-driven cases. (Same for PipelineScenarioExecution)